frontend/src/queries/schema.ts

```diff
@@ -2022,3 +2024,19 @@ export interface EventTaxonomyQuery extends DataNode<EventTaxonomyQueryResponse>
 export type EventTaxonomyQueryResponse = AnalyticsQueryResponseBase<EventTaxonomyResponse>
 
 export type CachedEventTaxonomyQueryResponse = CachedQueryResponse<EventTaxonomyQueryResponse>
+
+export interface ActorsPropertyTaxonomyResponse {
+    sample_values: string[]
+    sample_count: integer
+}
+
+export interface ActorsPropertyTaxonomyQuery extends DataNode<ActorsPropertyTaxonomyQueryResponse> {
+    kind: NodeKind.ActorTaxonomyQuery
+    type: PropertyDefinitionType.Group | PropertyDefinitionType.Person
+    property: string
+    group_type_index?: integer
```

Comment:
`type` is redundant, because `group_type_index` alone signifies the same. If `group_type_index` is present, then we're looking at a group type, otherwise we're looking at persons.